TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/PATCH-Positive-Basicinfo.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource     TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Teardown Keywords
+Force Tags      v2-api
+
+*** Variables ***
+${SUITE}          Core Metadata Device Profile PATCH Basicinfo Positive Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-deviceprofile-patch-basicinfo-positive.log
+
+*** Test Cases ***
+ProfilePATCH001 - Update one basicinfo of device profile
```

Comment:
use `ProfileBasicInfoPATCH001` because it should be easily distinguished with other profile cases